orttraining/orttraining/python/training/_ortmodule_graph_execution_manager.py

```diff
@@ -272,5 +272,10 @@ def _initialize_graph_builder(self, training):
         grad_builder_config.graph_transformer_config = C.GraphTransformerConfiguration()
         grad_builder_config.graph_transformer_config.propagate_cast_ops_level = self._propagate_cast_ops_level
         grad_builder_config.graph_transformer_config.propagate_cast_ops_allow = self._propagate_cast_ops_allow
+        grad_builder_config.loglevel = {_logger.LogLevel.VERBOSE : C.Severity.VERBOSE,
```

Comment:
Can we just pass the self._loglevel to the backend? After all, it just an integer.  
We can skip the pybind for "Severity" and make the frontend code simple and clean.

Reply:
I think this is more type safe and Thiago also prefers using typesafe data than int, I think.

Reply:
If we can make the onnxruntime Python API more idiomatic, it is preferable. Friendlier APIS make it more likely to be adopted by Python community users

Using ints to replace enums is kind of low level C API and might feel weird for Python users, especially the ones that never worked with C/C++ before